README.md

```diff
@@ -106,6 +106,7 @@ Chemfiles can read and write files in the following formats:
 - SERC Daresbury Laboratory CSSR files;
 - GROMACS formats: .gro;
 - TRIPOS mol2 files;
+- 
```

Comment:
Is this missing a `mmCIF` format ?

Reply:
I could've sworn I did that. Good catch...